sailfishusermanager.cpp

```diff
@@ -528,7 +538,11 @@ void SailfishUserManager::onCreatingJobFailed(SystemdManager::JobList &remaining
             // TODO: What to do?
             qCWarning(lcSUM) << "Unhandled error case";
         }
-    } // else nothing was done
+    } else { // nothing was done
+        qCWarning(lcSUM) << "User switching did not begin";
+        // TODO: This should signal that user switch was cancelled
```

Comment:
Could you log a task and add bug id ref here.

Reply:
Yes, created and added.